cpp/tests/fixed_point/fixed_point_tests.cu

```diff
@@ -514,4 +528,250 @@ TEST_F(FixedPointTest, DecimalXXThrustOnDevice)
   EXPECT_EQ(vec2, vec3);
 }
 
+template <typename T>
+using wrapper = cudf::test::fixed_width_column_wrapper<T>;
+
+TYPED_TEST(FixedPointTestBothReps, FixedPointSortedOrderGather)
```

Comment:
Generally when we add a new type we don't create a new test file that tests every single algorithm for that type. Instead, we have existing test files for each algorithm, and we augment those tests to use the new type.

Is there a reason you took this approach? In general these test only trivial data sizes, whereas the algorithm-specific tests are likely to do more comprehensive testing.

Reply:
I was thinking that at some point I would distribute the unit tests for the corresponding test files. For the time being it seemed easier to just keep them all in one file because:
1. I only have to run FIXED_POINT_TEST (instead of all the different tests)
2. I can easily reuse the unit test code without having to skip between files

At some point I can split them up though. Not meant to be permanent

Reply:
Adding a new type affects the whole library. Testing the new type using existing algorithm tests is a lot of testing, but it's necessary.  I would prefer that you add decimal32 and decimal64 to the the existing tests where they are supported.

Reply:
Done :slightly_smiling_face:

Reply:
Not really. :) What you did is move your new tests to the individual unit tests for each algorithm. What I asked is to *add fixed point* to the type lists used in the existing tests. E.g. run all the existing reductions tests on decimal32 and decima64, not add new special case tests for reductions on these types.

Reply:
Ahhh sorry ... I didn't understand your inital request. I add `decimal32` and `decimal64` to the type lists and see what happens.

Reply:
@harrism I have added `FixedPointTypes` which has been added to `FixedWidthTypes`. It is running for about 50% for tests :+1: For the others, I temporarily added `FixedWidthTypesWithoutFixedPoint` which can be slowly removed in follow-up PRs. A couple tests are currently failing but I will get those working tomorrow.